cfme/intelligence/chargeback.py

```diff
@@ -331,14 +278,16 @@ def am_i_here(self):
 
 @navigator.register(StorageRate, 'New')
 class StorageRateNew(CFMENavigateStep):
+    VIEW = AddStorageChargebackView
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        tb_select("Add a new Chargeback Rate")
+        self.view.configuration.item_select("Add a new Chargeback Rate")
 
 
 @navigator.register(StorageRate, 'Details')
 class StorageRateDetails(CFMENavigateStep):
+    VIEW = ChargebackView
```

Comment:
same view and am_i_here. I thought we are going to replace am_I_here by is_displayed in most places.